cirq/google/devices/known_devices_test.py

```diff
@@ -385,17 +386,17 @@ def test_json_dict():
         'constant': 'cirq.google.Bristlecone',
     }
 
-    from cirq.google.devices.known_devices import _NamedConstantXmonDevice
     with pytest.raises(ValueError, match='xmon device name'):
-        _NamedConstantXmonDevice._from_json_dict_('the_unknown_fiddler')
+        cgdk._NamedConstantXmonDevice._from_json_dict_('the_unknown_fiddler')
 
 
-def test_sycamore_device():
+@pytest.mark.parametrize('device', [cg.Sycamore, cgdk.Sycamore23])
+def test_sycamore_device(device):
     q0 = cirq.GridQubit(5, 4)
     q1 = cirq.GridQubit(5, 5)
     syc = cirq.FSimGate(theta=np.pi / 2, phi=np.pi / 6)(q0, q1)
     sqrt_iswap = cirq.FSimGate(theta=np.pi / 4, phi=0)(q0, q1)
-    cg.Sycamore.validate_operation(syc)
-    cg.Sycamore.validate_operation(sqrt_iswap)
+    device.validate_operation(syc)
+    device.validate_operation(sqrt_iswap)
```

Comment:
I think you need a test for invalid operations to catch the typos mentioned earlier.

Reply:
Done.  Thanks for the good catch!  That will teach me for hurrying out PRs as I am heading out for the evening!